### PR TITLE
ENH: Add html-swipe-response

### PR DESCRIPTION
**Pull request by richford**
This PR adds a new plugin, the HTMLSwipeResponse, which presents an HTML stimulus and allows the participant to provide a response either through a key press or by using a swipe gesture. It is ideal for two-alternative forced choice assessments that need to be responsive to both mobile and desktop browsers. It also includes an optional swipe animation which shows the stimulus being swiped offscreen.

### CODE REVIEW

**Comment by richford** [comment]
Tagging @AnyaWMa and @jyeatman for visibility.

**Comment by jyeatman** [comment]
super cool! Looking forward to checking this out once it's incorporated into ROAR-SWR

File: packages/plugin-html-swipe-response/package.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+  "name": "@jspsych-contrib/plugin-html-swipe-response",
+  "private": "true",
```
Comment (jodeleeuw) [inline packages/plugin-html-swipe-response/package.json:3]: I think the private flag will cause an issue with the release.

File: packages/plugin-html-swipe-response/package.json [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  "version": "1.0.0",
```
Comment (jodeleeuw) [inline packages/plugin-html-swipe-response/package.json:4]: The version should be 0.1 so that the changeset will auto bump to 1.0.

**Comment by jodeleeuw** [comment]
Love this idea. Thanks @richford ! The demo is great too ☺️

File: packages/plugin-html-swipe-response/package.json (hunk 2 above)
Comment (richford) [reply packages/plugin-html-swipe-response/package.json]: Thanks! Resolved in [aae94fa](https://github.com/jspsych/jspsych-contrib/pull/36/commits/aae94fa279443a90d2768c8b33e3c5c69c781ec1)

File: packages/plugin-html-swipe-response/package.json (hunk 1 above)
Comment (richford) [reply packages/plugin-html-swipe-response/package.json]: Thanks! Resolved in [aae94fa](https://github.com/jspsych/jspsych-contrib/pull/36/commits/aae94fa279443a90d2768c8b33e3c5c69c781ec1)

**Comment by jodeleeuw** [comment]
Can you add this one to the readme file as well?

**Comment by richford** [comment]
Sure thing! I did that in the latest commit. There will probably be a merge conflict after merging #35. I'm happy to rebase this PR onto that one if you'd like.

**Comment by jodeleeuw** [comment]
That would be great. Thanks!

**Comment by richford** [comment]
Thanks @jodeleeuw! I just rebased.

**Pull request merged**